<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Reservation</class>
 <widget class="QDialog" name="Reservation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="cbx_province">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="cbx_city">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>711</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>570</y>
     <width>62</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>711</width>
     <height>67</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="cbx_petname">
          <property name="currentText">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lb_hos_name">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEdit">
          <property name="maximumDate">
           <date>
            <year>2030</year>
            <month>12</month>
            <day>31</day>
           </date>
          </property>
          <property name="minimumDate">
           <date>
            <year>2022</year>
            <month>5</month>
            <day>1</day>
           </date>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbx_visit_time">
          <item>
           <property name="text">
            <string>11:00</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>11:30</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>12:00</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>12:30</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>13:00</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>13:30</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>14:00</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>14:30</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>15:00</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>15:30</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>16:00</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>16:30</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>17:00</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>17:30</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>18:00</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Symptom</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_symptom"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btn_reservation">
      <property name="maximumSize">
       <size>
        <width>104</width>
        <height>29</height>
       </size>
      </property>
      <property name="text">
       <string>RESERVATION</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
